core/controllers/acl_decorators.py

```diff
@@ -1119,6 +1119,55 @@ def test_can_voiceover(self, exploration_id, **kwargs):
     return test_can_voiceover
 
 
+def can_assign_voiceartist(handler):
+    """Decorator to check whether the user can assign voiceartist in
+    given exploration.
+
+    Args:
+        handler: function. The function to be decorated.
+
+    Returns:
+        function. The newly decorated function that now also checks if a user
+        has permission to assign voiceartist to a given exploration.
+    """
+
+    def test_can_assign_voiceartist(self, entity_type, entity_id, **kwargs):
```

Comment:
```suggestion
    def test_can_assign_voice_artist(self, entity_type, entity_id, **kwargs):
```

Follow this in all the places*

Reply:
Done